plugins/gssapi.c

```diff
@@ -1093,6 +1371,23 @@ gssapi_server_mech_authneg(context_t *text,
 	goto cleanup;
     }
 
+#ifdef GSS_USE_CC
+    /* Get ccache from sasl-config file if located elsewhere than /tmp */
```

Comment:
I do not understand the comment, in many systems ccache has been moved to different types (KEYRING, KCM, etc.. for quite some time, do you mean "if not the default ccache" ? If so I would phrase it that way, mentioning /tmp is confusing (even on systems where a FILE ccache is used the /tmp location has been dscouraged for a long time already)

Reply:
Heh :) Yeah, be gone with it :+1: